mypy.ini

```diff
@@ -0,0 +1,23 @@
+[mypy]
+ignore_errors = False
+warn_unused_configs = True
+warn_redundant_casts = True
+ignore_missing_imports = False
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+plugins = pydantic.mypy
+
+[mypy-sqlalchemy.*]
+ignore_missing_imports = True
+
+[mypy-deepdiff.*]
+ignore_missing_imports = True
+
+[mypy-felis.*]
```

Comment:
Are you sure this isn't meant to be a `lsst.sdm.tools.*` section? With the felis one looking something like

```ini
[mypy-lsst.felis.*]
ignore_missing_imports = False
ignore_errors = True
```